_format/1.2/index.md

```diff
@@ -93,6 +93,13 @@ values of the `ext` and `profile` parameters **MUST** equal a space-separated
 > specification requires that parameter values be surrounded by quotation marks
 > (U+0022 QUOTATION MARK, "\"").
 
+In the following example, two extensions and one profile have been applied:
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/vnd.api+json;ext="https://example.com/version https://example.com/bulk";profile="https://example.com/resource-timestamps"
+```
```

Comment:
I wasn't sure if the example should go here or a little bit later into the _Content Negotiation_ section. The specification addresses the topic at two places. Just before this new paragraph it defines that the value of the media type parameters is a space-separated list. But how and when the media type is used, is defined later in the section on content negotiation.

I'm not opinionated at all where we put this example.